hawc/apps/animalv2/managers.py

```diff
@@ -1,9 +1,30 @@
+from django.db.models import (
```

Comment:
here, and elsewhere update python imports to a single line instead of multiline; remove the final comma and then run ruff again

Reply:
done, think I got them all